PointOfDispensingApp/src/components/PatientCheckInComponent.vue

```diff
@@ -44,41 +59,51 @@
 </template>
 
 <script>
-  export default {
-    name: 'PatientCheckInComponent',
-    computed:
-    {
-       locationId() {
-        return this.$store.state.locationResource.locationId
-      },
-      locationName() {
-        return this.$store.state.locationResource.locationName
-      },
-      encounterStatus() {
-        return this.$store.state.encounterResource.encounterStatus
-      },
-      encounterTimeStamp() {
-        return this.$store.state.encounterResource.encounterTimeStamp
-      },
+import brokerRequests from "../brokerRequests";
+
+export default {
+  name: "PatientCheckInComponent",
+  computed: {
+    locationId() {
+      return this.$store.state.locationResource.locationId;
     },
-    methods: 
-    {
-      checkInComplete() {
-        //the following is sending dummy data until we have the API in place
-        const encounterResourcePayload = {
-          encounterStatus:'Arrived',
-          encounterTimeStamp: new Date().toISOString(),
-        }
-        //send data to Vuex
-        this.$store.dispatch('patientAdmitted', encounterResourcePayload)
-      }
+    locationName() {
+      return this.$store.state.locationResource.locationName;
     },
-    components: 
-    {
+    encounterStatus() {
+      return this.$store.state.encounterResource.encounterStatus;
+    },
+    encounterTimeStamp() {
+      return this.$store.state.encounterResource.encounterTimeStamp;
+    },
+  },
+  methods: {
+    checkInComplete() {
+      //the following is sending dummy data until we have the API in place
+      const encounterResourcePayload = {
+        encounterStatus: "Arrived",
+        encounterTimeStamp: new Date().toISOString(),
+      };
+      //send data to Vuex
+      this.$store.dispatch("patientAdmitted", encounterResourcePayload);
+
+      //Advance to the PatientHistory page
+      this.$router.push("PatientHistory");
```

Comment:
This is something you added, correct?  The Patient History page is something the user will likely check before selecting the "Patient information verified" button, so we do not need to auto advance to that page.  The more likely workflow will be that the user will go back to look up the next patient.
Recommendation:  do not automatically advance to a different page right now

Reply:
Done